shiny/ui/_page.py

```diff
@@ -85,6 +83,19 @@ def page_sidebar(
     if isinstance(title, str):
         title = tags.h1(title, class_="bslib-page-title")
 
+    # Extract sidebar from args
+    sidebars = [x for x in args if isinstance(x, Sidebar)]
```

Comment:
Make both tuples right away (without having to convert from list)
```suggestion
    sidebars = (x for x in args if isinstance(x, Sidebar))
    args = (x for x in args if not isinstance(x, Sidebar))
```